applications/machine_learning/src/modules/Kconfig.led_state

```diff
@@ -11,6 +11,28 @@ menuconfig ML_APP_LED_STATE
 
 if ML_APP_LED_STATE
 
+config ML_APP_LED_STATE_ANOMALY_THRESH
+	int "Anomaly threshold [in 0.001 units]"
+	default 600
+	help
```

Comment:
maybe adding a range for these configs?

Reply:
I wanted to avoid adding artificial limits here. The values may also depend on the used machine learning model.